pkg/hooks/hooks.go

```diff
@@ -143,6 +141,17 @@ func (h *HookDir) runHooks(dirpath string, hType HookType, pod Pod, podManifest
 }
 
 func (h *HookDir) RunHookType(hookType HookType, pod Pod, manifest *pods.Manifest) error {
-	dirpath := path.Join(h.dirpath, hookType.String())
-	return h.runHooks(dirpath, hookType, pod, manifest)
+	logger := h.logger.SubLogger(logrus.Fields{
+		"pod":      manifest.ID(),
+		"pod_path": pod.Path(),
+		"event":    hookType.String(),
+	})
+	logger.NoFields().Infoln("Running hooks")
```

Comment:
ah of course, that explains why we were using the infof to begin with. I'll put it back